src/server/HelixImportServer.js

```diff
@@ -136,9 +136,15 @@ export default class HelixServer extends EventEmitter {
     const respHeaders = ret.headers.plain();
     delete respHeaders['content-encoding'];
     delete respHeaders['content-length'];
-    delete respHeaders.location;
     respHeaders['access-control-allow-origin'] = '*';
 
+    if (respHeaders.location && !respHeaders.location.startsWith('/')) {
```

Comment:
don't you need to check for 301 or 302? otherwise a `200` with a `location` header will also rewrite.

Reply:
from the spec, the `location` header is only meaningful for 201 and 3xx.
In any case, it only makes the url relative which would be useful in any case (to avoid to have to rewrite the origin and use the `host` query param. It cannot hurt I think.